pkg/kogito/server.go

```diff
@@ -197,6 +199,24 @@ func devSandboxHandler() func(w http.ResponseWriter, r *http.Request) {
 		r.Host = r.URL.Host
 
 		proxy := httputil.NewSingleHostReverseProxy(targetUrl)
+
+		// tolerate self-signed certificates
+		proxy.Transport = &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
```

Comment:
Hi @ljmotta . Neither am I :-) But I realized that if we don't set this the default value will be `nil`. Anyway, this would only affect users that have (rely on) some proxy (to access the Internet) configured in the system environment (OS level). Not sure...